README.md

```diff
@@ -0,0 +1,39 @@
+# Hedvig Web Onboarding
+
+This is the web based onboarding that you can find at [https://www.hedvig.com/se-en/new-member](https://www.hedvig.com/se-en/new-member).
+
+## Prerequisites
+
+- `node`
+- `yarn`
+
+### Install Node with nvm
+
+1. Install `nvm` [https://github.com/creationix/nvm](https://github.com/creationix/nvm)
+2. `nvm install` (installs the node version specified in `.nvmrc`)
+3. `nvm use` (use version from `.nvmrc`)
+
+### Install Yarn
+
+1. Install `yarn` [https://classic.yarnpkg.com/en/docs/install/#mac-stable](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+
+#### Install Yarn with yvm
```

Comment:
wow this is so nice! But do you need to install yarn first to use `yvm`?

Reply:
Ah sorry, should maybe clarify that it's either or. I think `yvm` is nice but maybe more optional than `nvm`